pkg/test/framework/environments/kubernetes/flags.go

```diff
@@ -36,4 +36,6 @@ func init() {
 			"one test namespace in the system.")
 	flag.BoolVar(&globalSettings.DeployIstio, "istio.test.kube.deploy", globalSettings.DeployIstio,
 		"Deploy Istio into the target Kubernetes environment.")
+	flag.BoolVar(&globalSettings.NoIngress, "istio.test.kube.noingress", globalSettings.NoIngress,
```

Comment:
Let's not use negatives in the flag names, it gets confusing.  Maybe something like `useIngress`? Also, do you think it would be possible to test the ingress locally? In which case, it wouldn't be a kube-specific setting.

More generally, do we need this to be a flag at all? We already have a component, perhaps we could just include it as a dependency if we need it?